python_standards.md

```diff
@@ -0,0 +1,167 @@
+# Python Standards
```

Comment:
Please update the README to include these standards. I think it makes sense for both R and Python to be bullets nested under Scripts in the README.